### PR TITLE
404mk1

### PR DESCRIPTION
**Pull request by bkmac511**

### CODE REVIEW

File: apps/frontend/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:2]: `/styles/...` is global, don't use it
read how to use css modules
in this case, you should move the file to the same directory where 404.tsx is

File: apps/frontend/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <a className={styles.mainref} href="/">Strona Główna</a>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:16]: each `a` should have the same `className`
what's the point of creating a separate one for each?

File: apps/frontend/pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </li>
+
+        <li>
+          <a className={styles.news} href="/news">Aktualności</a>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:20]: wrap `<a>` in `next/link` - https://nextjs.org/docs/api-reference/next/link

tldr; we use that to make transition between pages smoother; always use it

File: apps/frontend/pages/404.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:8]: we don't use `h1, h2, h3, etc...` - check how to use `react-accesible-headings`

in this case, wrap an entire 404 in `<Level>` and use `<H>`

Comment (konhi) [inline apps/frontend/pages/404.tsx:8]:
```suggestion
      <h1 className={styles.header}><b>404.</b></h1>
```

File: apps/frontend/pages/404.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:14]:
```suggestion
      <ul className={styles.links}>
```

File: apps/frontend/pages/404.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:10]: why is it `<a`?

**Review by konhi — CHANGES_REQUESTED** [review]
